src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -300,6 +320,8 @@ class AddItemsPage extends Component {
       newItem: false,
       totalCount: 0,
       isFirstPageLoaded: false,
+      isDraftAvailable: false,
+      workflow: 'outbound',
```

Comment:
could we make it as enum, something like
```javascript
 const StockMovementDirection = {
   OUTBOUND: 'outbound',
   INBOUND: 'inbound',
 }
```

so it is "prepared" for other workflows and we don't hardcode strings if those will potentially be used in more places?